corehq/apps/app_manager/views/settings.py

```diff
@@ -72,9 +73,17 @@ def edit_commcare_profile(request, domain, app_id):
         for name, value in settings.get(settings_type, {}).items():
             if settings_type not in app.profile:
                 app.profile[settings_type] = {}
+
             app.profile[settings_type][name] = value
             changed[settings_type][name] = value
 
+    if "credentials" in changed["features"]:
```

Comment:
This part updates the `CredentialApplication` record's activity level based on the user's input.